scripts/3-getCreatedGameData.js

```diff
@@ -0,0 +1,29 @@
+import fetch from 'node-fetch';
+import fs from 'fs';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const BASE_URL = process.env.BASE_URL || '';
+
+const gameConfig = JSON.parse(fs.readFileSync('./gameConfig.json'));
+console.log(gameConfig);
```

Comment:
[Optional]

- [ ] It's best practice to remove `console.log` from the code (it's fine if you use it during development but you should remove it when you finish with it) 👍